lib/index.js

```diff
@@ -93,3 +96,82 @@ Klaviyo.prototype.track = function(track) {
     revenue: '$value'
   }));
 };
+
+/**
+ * Completed Order
+ *
+ * @param {Track} track
+ */
+
+Klaviyo.prototype.completedOrder = function(track) {
+  var products = track.products();
+  var categories = formatCategories(products);
+  var productNames = formatNames(products);
+  var items = formatItems(products);
```

Comment:
hahah yeah that's looks good. It was like this server side so i'll change it over there too.